navigation-fragments/src/main/java/dev/hotwire/navigation/navigator/Navigator.kt

```diff
@@ -43,7 +43,11 @@ class Navigator(
      */
     val currentDestination: HotwireDestination?
         get() = currentDialogDestination as? HotwireDestination
-            ?: host.childFragmentManager.primaryNavigationFragment as? HotwireDestination
+            ?: if (host.isAdded) {
```

Comment:
How about using the `NavigatorHost.isReady()` function, which is a bit more comprehensive?